ccd-definition/AuthorisationCaseField/AuthorisationCaseField.json

```diff
@@ -180,7 +180,8 @@
           },
       {
         "UserRoles": [
-          "caseworker-civil-admin"
+          "caseworker-civil-admin",
```

Comment:
Can we add these into the existing "UserRoles" for this field. Having 2 seperate sets looks odd and might lead to confusion